samples/multi/mix.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Multi.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :multi,
+      version: "0.1.0",
+      elixir: "~> 1.10",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Multi.Application, []}
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:config_cat, path: "../.."}
```

Comment:
`:configcat` now 😅

Reply:
I named this `config_cat` so that the module name would be `ConfigCat` and not `Configcat`. If we rename the application, then we have to go through and rename all of the modules.  This is Elixir naming conventions

Reply:
I know it's a convention but on all other SDKs they call it `ConfigCat`. I renamed it on #54, without changing too much of the internals... I don't mind calling the classes ConfigCat for the sake of consistency.